### ISSUE TITLE
Separation of hats ... (mailing list)

### ISSUE THREAD

**Issue by dims** [dapr/community#532]
- https://docs.dapr.io/operations/support/support-security-issues/ documents sending email to dapr@dapr.io
- https://github.com/dapr/community/blob/master/CODE-OF-CONDUCT.md documents sending email to codeofconduct@dapr.io
- There does not seem to be an email alias for private emails to STC itself.

Questions:
- Is there (can there be) a separate group of people responsible for CoCC?
- Should there be a separate alias for security? with its own set of people?
- How does one reach the leadership group (STC) privately for a non-security/non-cocc issue/question?

I know it's a small set of people wearing multiple hats right now and probably the same people looking at all the mailing lists, but if we set some blueprints now, we can encourage community folks to grow into these new titles/roles/positions as well. What do you think?

**Comment by dims** [comment]
xref: https://github.com/cncf/toc/issues/1354

**Comment by msfussell** [comment]
> * Is there (can there be) a separate group of people responsible for CoCC?
Agreed. Added maintainers (not in STC) to the CoCC email list above.


> * Should there be a separate alias for security? with its own set of people?
Created security@dapr.io and added maintainers/approvers focused on security. Also update here https://docs.dapr.io/operations/support/support-security-issues/#reporting-process to reflect this change.


> * How does one reach the leadership group (STC) privately for a non-security/non-cocc issue/question?
There is an stc@dapr.io email alias that includes the STC members. I have added this here to the getting in touch section https://github.com/dapr/community/blob/master/steering-and-technical-committee-charter.md#getting-in-touch
